storage_controller/Cargo.toml

```diff
@@ -50,6 +50,7 @@ diesel = { version = "2.2.6", features = [
     "chrono",
 ] }
 diesel_migrations = { version = "2.2.0" }
+pq-sys = { version = "0.6", features = ["bundled"]}
```

Comment:
Before merging, we should discuss it with Storage team

Reply:
And in that case. we probably should undo the change made by 39be2b0108.